src/jekyll/chirpy.ts

```diff
@@ -52,11 +52,26 @@ export async function convertToChirpy(plugin: O2Plugin) {
         .converting(await plugin.app.vault.read(file));
 
       await plugin.app.vault.modify(file, result);
-      await moveFiles(
-        `${vaultAbsolutePath(plugin)}/${plugin.obsidianPathSettings.readyFolder}`,
-        settings.targetPath(),
-        settings.pathReplacer,
-      ).then(() => new Notice('Moved files to Chirpy successfully.', 5000));
+
+      // Where can I create this code block(create a function)?
+      const path: string = file.path;
+      const directory = path.substring(0, path.lastIndexOf('/'));
+      const folder = directory.substring(directory.lastIndexOf('/') + 1);
+      // end code block
+
+      if (folder !== plugin.obsidianPathSettings.readyFolder) {
+        await moveFiles(
+          `${vaultAbsolutePath(plugin)}/${plugin.obsidianPathSettings.readyFolder}/${folder}`,
+          settings.targetSubPath(folder),
+          settings.pathReplacer,
+        ).then(() => new Notice('Moved files to Chirpy successfully.', 5000));
+      } else {
+        await moveFiles(
+          `${vaultAbsolutePath(plugin)}/${plugin.obsidianPathSettings.readyFolder}`,
+          settings.targetPath(),
+          settings.pathReplacer,
+        ).then(() => new Notice('Moved files to Chirpy successfully.', 5000));
+      }
```

Comment:
Since you'll be doing a refactoring later, I believe it's not necessary for now, and I think this proposed solution is excellent to avoid rework in the future. Congratulations on the code! :)

I'll remove the comments in the next commit.